src/iop/filmicrgb.c

```diff
@@ -4248,10 +4248,8 @@ void gui_init(dt_iop_module_t *self)
                                                     "useful to give a safety margin to extreme luminances."));
 
   // Auto tune slider
-  GtkWidget *hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 0);
-  gtk_box_pack_start(GTK_BOX(hbox), dt_ui_label_new(_("auto tune levels")), TRUE, TRUE, 0);
-  g->auto_button = dt_color_picker_new(self, DT_COLOR_PICKER_AREA, NULL);
-  gtk_box_pack_start(GTK_BOX(hbox), g->auto_button, FALSE, FALSE, 0);
+  g->auto_button = dt_color_picker_new(self, DT_COLOR_PICKER_AREA, dt_bauhaus_combobox_new(self));
+  dt_bauhaus_widget_set_label(g->auto_button, NULL, N_("auto tune levels"));
   dt_action_define_iop(self, NULL, N_("auto tune levels"), GTK_WIDGET(g->auto_button), &dt_action_def_toggle);
```

Comment:
Please also remove this `dt_action_define_iop`; the shortcut is already fully set up by `dt_bauhaus_widget_set_label`.

Normally, buttons "attached to" a combo need to be called with a shortcut that has _element_ "button". But in this case it makes sense to not require that, for easier, more sensible, setup, but also to not break shortcuts for people that now have them set up via the button.

Please change
https://github.com/darktable-org/darktable/blob/26c52b36b5e5371488b9ec9ed7d156afad5077f1/src/bauhaus/bauhaus.c#L3420
and
https://github.com/darktable-org/darktable/blob/26c52b36b5e5371488b9ec9ed7d156afad5077f1/src/bauhaus/bauhaus.c#L3456
to
`if(element == DT_ACTION_ELEMENT_BUTTON || !w->data.combobox.entries->len)`

Reply:
Thanks. Should be fixed on last commit.

Reply:
Haven't tested, but looks good.